.conan2/profiles/gcc8x86

```diff
@@ -0,0 +1,26 @@
+{% set standalone_toolchain = "/opt/x-tools/i686-gcc8x86-linux-gnu" %}
+{% set target_host = "i686-gcc8x86-linux-gnu" %}
+{% set cc_compiler = "gcc" %}
+{% set cxx_compiler = "g++" %}
+
+[settings]
+arch=x86
+os=Linux
+compiler=gcc
+compiler.version=8
+compiler.libcxx=libstdc++11
+build_type=RelWithDebInfo
+
+[buildenv]
+PATH=+(path){{ standalone_toolchain }}/bin
+CHOST={{ target_host }}
+AR={{ target_host }}-ar
+AS={{ target_host }}-as
+RANLIB={{ target_host }}-ranlib
+CC={{ target_host }}-{{ cc_compiler }}
+CXX={{ target_host }}-{{ cxx_compiler }}
+LD={{ target_host }}-ld
+STRIP={{ target_host }}-strip
+
+[conf]
+tools.cmake.cmaketoolchain:user_toolchain = ["{{ standalone_toolchain }}/toolchain.cmake"]
```

Comment:
@prince-chrismc
Would be interesting if this profile looks good to you for cross-compiling scenarios.
Especially appending the `PATH` + setting the environment variables and appending the conan `user_toolchain` with our's.

_Compared to our old conan v1 profile:_
I had to **append** the `PATH` instead of  replacing it with the toolchain binary-path, otherwise the cmake executable couldn't be found. Removing the `PATH` is not possible, due to autotools (for building libxml2 dependency) searching for the toolchains gcc `i686-gcc8x86-linux-gnu-gcc`

Reply:
You're best bet is to open an issue against the main repo and get an answer from the team :)

Reply:
https://github.com/conan-io/conan/issues/14686#issue-1884090631